src/Features/LanguageServer/ProtocolUnitTests/Completion/CompletionTests.cs

```diff
@@ -57,13 +57,13 @@ void M()
             var document = testLspServer.GetCurrentSolution().Projects.First().Documents.First();
 
             var expected = await CreateCompletionItemAsync(label: "A", kind: LSP.CompletionItemKind.Class, tags: new string[] { "Class", "Internal" },
-                request: completionParams, document: document, commitCharacters: CompletionRules.Default.DefaultCommitCharacters, insertText: "A").ConfigureAwait(false);
+                request: completionParams, document: document, commitCharacters: CompletionRules.Default.DefaultCommitCharacters).ConfigureAwait(false);
```

Comment:
completion list is serialized into an optimized completion list which removes duplicate data in the insert text.  So when we deserialize in the test it will be null